WordPressAuthenticator/Unified Auth/View Related/Site Address/SiteAddressViewController.swift

```diff
@@ -140,6 +140,17 @@ final class SiteAddressViewController: LoginViewController {
         }
     }
 
+    override func displayRemoteError(_ error: Error) {
+        guard authenticationDelegate.shouldHandleError(error) == true else {
```

Comment:
Can this just be `guard authenticationDelegate.shouldHandleError(error) else {`?

Reply:
iirc, this is part of Woo's coding style. I could be wrong though.

Reply:
Yes, it's probably muscle memory. I'll update it